lib/tasks/import.rake

```diff
@@ -1,4 +1,10 @@
-namespace :db do
+namespace :pqa do
+  desc "Perform nightly import"
+  task :nightly_import, [] => :environment do
+    report = ImportWorker.new.perform
+    puts report.inspect
```

Comment:
cron jobs should be silent unless there is an error. If you want these logs to be somewhere then they should be written to a log file (and ideally this should be *structured data* - i.e. written as JSON via LogStuff.)